app/lib/BundlableLabelableBaseModelWithAttributes.php

```diff
@@ -722,8 +722,14 @@ public function set($pa_fields, $pm_value="", $pa_options=null) {
 		}
 		
 		if ($this->opo_idno_plugin_instance) {
+			if ($this->isChild()) {
```

Comment:
This actually passes the parent idno to isChild instead of setting the parent_value to null (Original code did not pass a second argument). 

Using self::getIdnoForID() is consistent with what happens if you call isChild (on this class, not the multipartIDNumber version we are actually calling here)